REST/src/main/java/pl/jakubowskiprzemyslaw/REST/utils/SendMessage.java

```diff
@@ -0,0 +1,18 @@
+package pl.jakubowskiprzemyslaw.REST.utils;
+
+import org.springframework.messaging.simp.SimpMessagingTemplate;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SendMessage {
```

Comment:
MessageSender

Reply:
it's done already, will be shown in next PR